test/src_test/include/test_vinecop_class.hpp

```diff
@@ -406,9 +411,9 @@ TEST_F(VinecopTest, sparse_both_selection)
   controls.set_select_threshold(true);
   // controls.set_show_trace(true);
   Vinecop fit(7);
-  fit.select_all(u, controls);
+  fit.select(u, controls);
   EXPECT_NEAR(fit.get_loglik(), fit.loglik(u), 0.001);
-  fit.select_families(u, controls);
+  fit.select(u, controls);
   EXPECT_NEAR(fit.get_loglik(), fit.loglik(u), 0.001);
 }
 }
```

Comment:
Should we have a dedicated unit test for partial selection (checking that the specified trees are actually in the final model)?

Reply:
My thought was that I really replicated the tests that we had before, plus I added the test that the partial selection behaves as expected in `fixed_truncation`. Not sure what we would gain from the dedicated unit test, but I'm open if you can make a case!

Reply:
If I understand correctly, the unit tests currently only show that we can do what we did before without bugs. But I don't think that the new feature is tested, i.e. if I specify only the first 4 trees, does the fitted model actually contain those first 4 trees + more?

Reply:
This is tested here: https://github.com/vinecopulib/vinecopulib/pull/457/files#diff-3d4a12377af54307871de500e34d1849R372-R373

Reply:
https://github.com/vinecopulib/vinecopulib/pull/457/commits/80eaf29cde9a7cbc525c51b421aeaab1b38e26b1